[2025-08-19T09:17:26.756+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-19T09:17:26.950+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.transform_golden manual__2025-08-19T09:11:07.475998+00:00 [queued]>
[2025-08-19T09:17:27.001+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.transform_golden manual__2025-08-19T09:11:07.475998+00:00 [queued]>
[2025-08-19T09:17:27.002+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 4
[2025-08-19T09:17:27.148+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): transform_golden> on 2025-08-19 09:11:07.475998+00:00
[2025-08-19T09:17:27.194+0000] {standard_task_runner.py:63} INFO - Started process 2219 to run task
[2025-08-19T09:17:27.215+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'youtube_pipeline_batch_data', 'transform_golden', 'manual__2025-08-19T09:11:07.475998+00:00', '--job-id', '597', '--raw', '--subdir', 'DAGS_FOLDER/youtube_pipeline.py', '--cfg-path', '/tmp/tmpsmlhbtub']
[2025-08-19T09:17:27.232+0000] {standard_task_runner.py:91} INFO - Job 597: Subtask transform_golden
[2025-08-19T09:17:27.571+0000] {task_command.py:426} INFO - Running <TaskInstance: youtube_pipeline_batch_data.transform_golden manual__2025-08-19T09:11:07.475998+00:00 [running]> on host 54bf3a3eaeab
[2025-08-19T09:17:27.731+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='youtube_pipeline_batch_data' AIRFLOW_CTX_TASK_ID='transform_golden' AIRFLOW_CTX_EXECUTION_DATE='2025-08-19T09:11:07.475998+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-19T09:11:07.475998+00:00'
[2025-08-19T09:17:27.732+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-19T09:17:27.757+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-19T09:17:27.758+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'docker exec spark-master spark-submit --master spark://spark-master:7077 --deploy-mode client --packages org.apache.spark:spark-avro_2.12:3.5.1 /opt/***/jobs/transformation/transform_to_golden_dataset.py']
[2025-08-19T09:17:27.772+0000] {subprocess.py:86} INFO - Output:
[2025-08-19T09:17:36.237+0000] {subprocess.py:93} INFO - :: loading settings :: url = jar:file:/opt/bitnami/spark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-08-19T09:17:36.385+0000] {subprocess.py:93} INFO - Ivy Default Cache set to: /opt/bitnami/spark/.ivy2/cache
[2025-08-19T09:17:36.387+0000] {subprocess.py:93} INFO - The jars for the packages stored in: /opt/bitnami/spark/.ivy2/jars
[2025-08-19T09:17:36.397+0000] {subprocess.py:93} INFO - org.apache.spark#spark-avro_2.12 added as a dependency
[2025-08-19T09:17:36.399+0000] {subprocess.py:93} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-90fbe196-1980-4ebc-92de-e5bb71722715;1.0
[2025-08-19T09:17:36.400+0000] {subprocess.py:93} INFO - 	confs: [default]
[2025-08-19T09:17:36.849+0000] {subprocess.py:93} INFO - 	found org.apache.spark#spark-avro_2.12;3.5.1 in central
[2025-08-19T09:17:36.906+0000] {subprocess.py:93} INFO - 	found org.tukaani#xz;1.9 in central
[2025-08-19T09:17:36.929+0000] {subprocess.py:93} INFO - :: resolution report :: resolve 521ms :: artifacts dl 9ms
[2025-08-19T09:17:36.929+0000] {subprocess.py:93} INFO - 	:: modules in use:
[2025-08-19T09:17:36.930+0000] {subprocess.py:93} INFO - 	org.apache.spark#spark-avro_2.12;3.5.1 from central in [default]
[2025-08-19T09:17:36.930+0000] {subprocess.py:93} INFO - 	org.tukaani#xz;1.9 from central in [default]
[2025-08-19T09:17:36.931+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-08-19T09:17:36.933+0000] {subprocess.py:93} INFO - 	|                  |            modules            ||   artifacts   |
[2025-08-19T09:17:36.933+0000] {subprocess.py:93} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-08-19T09:17:36.934+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-08-19T09:17:36.934+0000] {subprocess.py:93} INFO - 	|      default     |   2   |   0   |   0   |   0   ||   2   |   0   |
[2025-08-19T09:17:36.934+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-08-19T09:17:36.935+0000] {subprocess.py:93} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-90fbe196-1980-4ebc-92de-e5bb71722715
[2025-08-19T09:17:36.937+0000] {subprocess.py:93} INFO - 	confs: [default]
[2025-08-19T09:17:36.942+0000] {subprocess.py:93} INFO - 	0 artifacts copied, 2 already retrieved (0kB/7ms)
[2025-08-19T09:17:37.952+0000] {subprocess.py:93} INFO - 25/08/19 09:17:37 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-08-19T09:17:41.454+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO SparkContext: Running Spark version 3.5.1
[2025-08-19T09:17:41.461+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO SparkContext: OS info Linux, 6.11.0-29-generic, amd64
[2025-08-19T09:17:41.465+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO SparkContext: Java version 17.0.12
[2025-08-19T09:17:41.533+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO ResourceUtils: ==============================================================
[2025-08-19T09:17:41.536+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO ResourceUtils: No custom resources configured for spark.driver.
[2025-08-19T09:17:41.539+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO ResourceUtils: ==============================================================
[2025-08-19T09:17:41.542+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO SparkContext: Submitted application: Processed Zone - Golden Dataset
[2025-08-19T09:17:41.598+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2025-08-19T09:17:41.621+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO ResourceProfile: Limiting resource is cpu
[2025-08-19T09:17:41.623+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2025-08-19T09:17:41.796+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO SecurityManager: Changing view acls to: spark
[2025-08-19T09:17:41.796+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO SecurityManager: Changing modify acls to: spark
[2025-08-19T09:17:41.797+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO SecurityManager: Changing view acls groups to:
[2025-08-19T09:17:41.803+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO SecurityManager: Changing modify acls groups to:
[2025-08-19T09:17:41.803+0000] {subprocess.py:93} INFO - 25/08/19 09:17:41 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: spark; groups with view permissions: EMPTY; users with modify permissions: spark; groups with modify permissions: EMPTY
[2025-08-19T09:17:44.261+0000] {subprocess.py:93} INFO - 25/08/19 09:17:44 INFO Utils: Successfully started service 'sparkDriver' on port 39895.
[2025-08-19T09:17:44.330+0000] {subprocess.py:93} INFO - 25/08/19 09:17:44 INFO SparkEnv: Registering MapOutputTracker
[2025-08-19T09:17:44.399+0000] {subprocess.py:93} INFO - 25/08/19 09:17:44 INFO SparkEnv: Registering BlockManagerMaster
[2025-08-19T09:17:44.423+0000] {subprocess.py:93} INFO - 25/08/19 09:17:44 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2025-08-19T09:17:44.424+0000] {subprocess.py:93} INFO - 25/08/19 09:17:44 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2025-08-19T09:17:44.431+0000] {subprocess.py:93} INFO - 25/08/19 09:17:44 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2025-08-19T09:17:44.460+0000] {subprocess.py:93} INFO - 25/08/19 09:17:44 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-c35d0527-94b4-44fb-bde0-3b21ff1bc78b
[2025-08-19T09:17:44.481+0000] {subprocess.py:93} INFO - 25/08/19 09:17:44 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2025-08-19T09:17:44.508+0000] {subprocess.py:93} INFO - 25/08/19 09:17:44 INFO SparkEnv: Registering OutputCommitCoordinator
[2025-08-19T09:17:44.859+0000] {subprocess.py:93} INFO - 25/08/19 09:17:44 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2025-08-19T09:17:45.081+0000] {subprocess.py:93} INFO - 25/08/19 09:17:45 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2025-08-19T09:17:45.195+0000] {subprocess.py:93} INFO - 25/08/19 09:17:45 INFO SparkContext: Added JAR file:///opt/bitnami/spark/.ivy2/jars/org.apache.spark_spark-avro_2.12-3.5.1.jar at spark://b208641feb52:39895/jars/org.apache.spark_spark-avro_2.12-3.5.1.jar with timestamp 1755595061391
[2025-08-19T09:17:45.197+0000] {subprocess.py:93} INFO - 25/08/19 09:17:45 INFO SparkContext: Added JAR file:///opt/bitnami/spark/.ivy2/jars/org.tukaani_xz-1.9.jar at spark://b208641feb52:39895/jars/org.tukaani_xz-1.9.jar with timestamp 1755595061391
[2025-08-19T09:17:45.204+0000] {subprocess.py:93} INFO - 25/08/19 09:17:45 INFO SparkContext: Added file file:///opt/bitnami/spark/.ivy2/jars/org.apache.spark_spark-avro_2.12-3.5.1.jar at spark://b208641feb52:39895/files/org.apache.spark_spark-avro_2.12-3.5.1.jar with timestamp 1755595061391
[2025-08-19T09:17:45.211+0000] {subprocess.py:93} INFO - 25/08/19 09:17:45 INFO Utils: Copying /opt/bitnami/spark/.ivy2/jars/org.apache.spark_spark-avro_2.12-3.5.1.jar to /tmp/spark-4efd3942-35f0-4efc-871e-ce789dbc7c41/userFiles-26fa99ab-e6e7-437b-94af-adb18e8d53de/org.apache.spark_spark-avro_2.12-3.5.1.jar
[2025-08-19T09:17:45.268+0000] {subprocess.py:93} INFO - 25/08/19 09:17:45 INFO SparkContext: Added file file:///opt/bitnami/spark/.ivy2/jars/org.tukaani_xz-1.9.jar at spark://b208641feb52:39895/files/org.tukaani_xz-1.9.jar with timestamp 1755595061391
[2025-08-19T09:17:45.271+0000] {subprocess.py:93} INFO - 25/08/19 09:17:45 INFO Utils: Copying /opt/bitnami/spark/.ivy2/jars/org.tukaani_xz-1.9.jar to /tmp/spark-4efd3942-35f0-4efc-871e-ce789dbc7c41/userFiles-26fa99ab-e6e7-437b-94af-adb18e8d53de/org.tukaani_xz-1.9.jar
[2025-08-19T09:17:46.000+0000] {subprocess.py:93} INFO - 25/08/19 09:17:45 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2025-08-19T09:17:46.256+0000] {subprocess.py:93} INFO - 25/08/19 09:17:46 INFO TransportClientFactory: Successfully created connection to spark-master/172.18.0.4:7077 after 145 ms (0 ms spent in bootstraps)
[2025-08-19T09:17:46.744+0000] {subprocess.py:93} INFO - 25/08/19 09:17:46 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20250819091746-0006
[2025-08-19T09:17:46.776+0000] {subprocess.py:93} INFO - 25/08/19 09:17:46 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 43727.
[2025-08-19T09:17:46.778+0000] {subprocess.py:93} INFO - 25/08/19 09:17:46 INFO NettyBlockTransferService: Server created on b208641feb52:43727
[2025-08-19T09:17:46.783+0000] {subprocess.py:93} INFO - 25/08/19 09:17:46 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2025-08-19T09:17:46.796+0000] {subprocess.py:93} INFO - 25/08/19 09:17:46 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, b208641feb52, 43727, None)
[2025-08-19T09:17:46.808+0000] {subprocess.py:93} INFO - 25/08/19 09:17:46 INFO BlockManagerMasterEndpoint: Registering block manager b208641feb52:43727 with 434.4 MiB RAM, BlockManagerId(driver, b208641feb52, 43727, None)
[2025-08-19T09:17:46.813+0000] {subprocess.py:93} INFO - 25/08/19 09:17:46 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, b208641feb52, 43727, None)
[2025-08-19T09:17:46.821+0000] {subprocess.py:93} INFO - 25/08/19 09:17:46 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, b208641feb52, 43727, None)
[2025-08-19T09:17:47.300+0000] {subprocess.py:93} INFO - 25/08/19 09:17:47 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2025-08-19T09:17:48.206+0000] {subprocess.py:93} INFO - 25/08/19 09:17:48 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2025-08-19T09:17:48.221+0000] {subprocess.py:93} INFO - 25/08/19 09:17:48 INFO SharedState: Warehouse path is 'file:/opt/bitnami/spark/spark-warehouse'.
[2025-08-19T09:17:52.279+0000] {subprocess.py:93} INFO - 25/08/19 09:17:52 INFO InMemoryFileIndex: It took 216 ms to list leaf files for 1 paths.
[2025-08-19T09:17:56.289+0000] {subprocess.py:93} INFO - 25/08/19 09:17:56 INFO InMemoryFileIndex: It took 25 ms to list leaf files for 1 paths.
[2025-08-19T09:18:03.413+0000] {subprocess.py:93} INFO - 25/08/19 09:18:03 INFO InMemoryFileIndex: It took 17 ms to list leaf files for 1 paths.
[2025-08-19T09:18:03.574+0000] {subprocess.py:93} INFO - 25/08/19 09:18:03 INFO InMemoryFileIndex: It took 34 ms to list leaf files for 1 paths.
[2025-08-19T09:18:08.249+0000] {subprocess.py:93} INFO - 25/08/19 09:18:08 INFO InMemoryFileIndex: It took 15 ms to list leaf files for 1 paths.
[2025-08-19T09:18:08.314+0000] {subprocess.py:93} INFO - 25/08/19 09:18:08 INFO InMemoryFileIndex: It took 14 ms to list leaf files for 1 paths.
[2025-08-19T09:18:11.338+0000] {subprocess.py:93} INFO - 25/08/19 09:18:11 INFO InMemoryFileIndex: It took 112 ms to list leaf files for 1 paths.
[2025-08-19T09:18:11.637+0000] {subprocess.py:93} INFO - 25/08/19 09:18:11 INFO InMemoryFileIndex: It took 70 ms to list leaf files for 1 paths.
[2025-08-19T09:18:13.805+0000] {subprocess.py:93} INFO - 25/08/19 09:18:13 INFO InMemoryFileIndex: It took 50 ms to list leaf files for 1 paths.
[2025-08-19T09:18:14.002+0000] {subprocess.py:93} INFO - 25/08/19 09:18:14 INFO InMemoryFileIndex: It took 47 ms to list leaf files for 1 paths.
[2025-08-19T09:18:16.777+0000] {subprocess.py:93} INFO - 25/08/19 09:18:16 INFO InMemoryFileIndex: It took 36 ms to list leaf files for 1 paths.
[2025-08-19T09:18:16.866+0000] {subprocess.py:93} INFO - 25/08/19 09:18:16 INFO InMemoryFileIndex: It took 8 ms to list leaf files for 1 paths.
[2025-08-19T09:18:17.984+0000] {subprocess.py:93} INFO - 25/08/19 09:18:17 INFO InMemoryFileIndex: It took 19 ms to list leaf files for 1 paths.
[2025-08-19T09:18:18.264+0000] {subprocess.py:93} INFO - 25/08/19 09:18:18 INFO InMemoryFileIndex: It took 90 ms to list leaf files for 1 paths.
[2025-08-19T09:18:19.247+0000] {subprocess.py:93} INFO - 25/08/19 09:18:19 INFO InMemoryFileIndex: It took 8 ms to list leaf files for 1 paths.
[2025-08-19T09:18:19.291+0000] {subprocess.py:93} INFO - 25/08/19 09:18:19 INFO InMemoryFileIndex: It took 9 ms to list leaf files for 1 paths.
[2025-08-19T09:18:20.786+0000] {subprocess.py:93} INFO - 25/08/19 09:18:20 INFO InMemoryFileIndex: It took 31 ms to list leaf files for 1 paths.
[2025-08-19T09:18:20.937+0000] {subprocess.py:93} INFO - 25/08/19 09:18:20 INFO InMemoryFileIndex: It took 28 ms to list leaf files for 1 paths.
[2025-08-19T09:18:21.771+0000] {subprocess.py:93} INFO - 25/08/19 09:18:21 INFO InMemoryFileIndex: It took 53 ms to list leaf files for 1 paths.
[2025-08-19T09:18:22.171+0000] {subprocess.py:93} INFO - 25/08/19 09:18:22 INFO InMemoryFileIndex: It took 84 ms to list leaf files for 1 paths.
[2025-08-19T09:18:31.188+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:18:31.195+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#7),isnotnull(CASE WHEN (length(trim(regexp_replace(title#2, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#2, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#2, [^\x20-\x7E], , 1), None) END),(cast(views#7 as bigint) > 0)
[2025-08-19T09:18:31.224+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:18:31.225+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#32)
[2025-08-19T09:18:31.267+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:18:31.269+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#500),isnotnull(CASE WHEN (length(trim(regexp_replace(title#495, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#495, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#495, [^\x20-\x7E], , 1), None) END),(cast(views#500 as bigint) > 0)
[2025-08-19T09:18:31.270+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:18:31.271+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#525)
[2025-08-19T09:18:31.378+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:18:31.394+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#993),isnotnull(CASE WHEN (length(trim(regexp_replace(title#988, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#988, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#988, [^\x20-\x7E], , 1), None) END),(cast(views#993 as bigint) > 0)
[2025-08-19T09:18:31.406+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:18:31.413+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#1018)
[2025-08-19T09:18:31.465+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:18:31.470+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#1486),isnotnull(CASE WHEN (length(trim(regexp_replace(title#1481, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#1481, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#1481, [^\x20-\x7E], , 1), None) END),(cast(views#1486 as bigint) > 0)
[2025-08-19T09:18:31.475+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:18:31.478+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#1511)
[2025-08-19T09:18:31.507+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:18:31.509+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#1979),isnotnull(CASE WHEN (length(trim(regexp_replace(title#1974, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#1974, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#1974, [^\x20-\x7E], , 1), None) END),(cast(views#1979 as bigint) > 0)
[2025-08-19T09:18:31.511+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:18:31.512+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#2004)
[2025-08-19T09:18:31.535+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:18:31.536+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#2472),isnotnull(CASE WHEN (length(trim(regexp_replace(title#2467, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#2467, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#2467, [^\x20-\x7E], , 1), None) END),(cast(views#2472 as bigint) > 0)
[2025-08-19T09:18:31.543+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:18:31.546+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#2497)
[2025-08-19T09:18:31.605+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:18:31.607+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#2965),isnotnull(CASE WHEN (length(trim(regexp_replace(title#2960, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#2960, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#2960, [^\x20-\x7E], , 1), None) END),(cast(views#2965 as bigint) > 0)
[2025-08-19T09:18:31.608+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:18:31.608+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#2990)
[2025-08-19T09:18:31.622+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:18:31.624+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#3458),isnotnull(CASE WHEN (length(trim(regexp_replace(title#3453, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#3453, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#3453, [^\x20-\x7E], , 1), None) END),(cast(views#3458 as bigint) > 0)
[2025-08-19T09:18:31.626+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:18:31.626+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#3483)
[2025-08-19T09:18:31.661+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:18:31.664+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#3951),isnotnull(CASE WHEN (length(trim(regexp_replace(title#3946, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#3946, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#3946, [^\x20-\x7E], , 1), None) END),(cast(views#3951 as bigint) > 0)
[2025-08-19T09:18:31.666+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:18:31.669+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#3976)
[2025-08-19T09:18:31.712+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(views)
[2025-08-19T09:18:31.718+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(views#4444),isnotnull(CASE WHEN (length(trim(regexp_replace(title#4439, [^\x20-\x7E], , 1), None)) > 100) THEN concat(substring(trim(regexp_replace(title#4439, [^\x20-\x7E], , 1), None), 1, 97), ...) ELSE trim(regexp_replace(title#4439, [^\x20-\x7E], , 1), None) END),(cast(views#4444 as bigint) > 0)
[2025-08-19T09:18:31.722+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Pushed Filters: IsNotNull(category_id)
[2025-08-19T09:18:31.723+0000] {subprocess.py:93} INFO - 25/08/19 09:18:31 INFO FileSourceStrategy: Post-Scan Filters: isnotnull(category_id#4469)
[2025-08-19T09:18:34.178+0000] {subprocess.py:93} INFO - 25/08/19 09:18:34 WARN SparkStringUtils: Truncated the string representation of a plan since it was too large. This behavior can be adjusted by setting 'spark.sql.debug.maxToStringFields'.
[2025-08-19T09:18:39.395+0000] {subprocess.py:93} INFO - 25/08/19 09:18:39 INFO CodeGenerator: Code generated in 2826.861533 ms
[2025-08-19T09:18:39.437+0000] {subprocess.py:93} INFO - 25/08/19 09:18:39 INFO CodeGenerator: Code generated in 2822.895863 ms
[2025-08-19T09:18:39.444+0000] {subprocess.py:93} INFO - 25/08/19 09:18:39 INFO CodeGenerator: Code generated in 2851.545467 ms
[2025-08-19T09:18:39.452+0000] {subprocess.py:93} INFO - 25/08/19 09:18:39 INFO CodeGenerator: Code generated in 2840.959268 ms
[2025-08-19T09:18:39.456+0000] {subprocess.py:93} INFO - 25/08/19 09:18:39 INFO CodeGenerator: Code generated in 2833.036025 ms
[2025-08-19T09:18:39.457+0000] {subprocess.py:93} INFO - 25/08/19 09:18:39 INFO CodeGenerator: Code generated in 2796.006693 ms
[2025-08-19T09:18:39.470+0000] {subprocess.py:93} INFO - 25/08/19 09:18:39 INFO CodeGenerator: Code generated in 2804.905226 ms
[2025-08-19T09:18:39.477+0000] {subprocess.py:93} INFO - 25/08/19 09:18:39 INFO CodeGenerator: Code generated in 2837.787491 ms
[2025-08-19T09:18:39.480+0000] {subprocess.py:93} INFO - 25/08/19 09:18:39 INFO CodeGenerator: Code generated in 2827.918461 ms
[2025-08-19T09:18:39.493+0000] {subprocess.py:93} INFO - 25/08/19 09:18:39 INFO CodeGenerator: Code generated in 2903.66311 ms
[2025-08-19T09:18:40.050+0000] {subprocess.py:93} INFO - 25/08/19 09:18:40 INFO MemoryStore: Block broadcast_5 stored as values in memory (estimated size 200.9 KiB, free 434.2 MiB)
[2025-08-19T09:18:40.054+0000] {subprocess.py:93} INFO - 25/08/19 09:18:40 INFO MemoryStore: Block broadcast_8 stored as values in memory (estimated size 200.9 KiB, free 434.0 MiB)
[2025-08-19T09:18:40.073+0000] {subprocess.py:93} INFO - 25/08/19 09:18:40 INFO MemoryStore: Block broadcast_0 stored as values in memory (estimated size 200.9 KiB, free 433.8 MiB)
[2025-08-19T09:18:40.073+0000] {subprocess.py:93} INFO - 25/08/19 09:18:40 INFO MemoryStore: Block broadcast_7 stored as values in memory (estimated size 200.9 KiB, free 433.6 MiB)
[2025-08-19T09:18:40.087+0000] {subprocess.py:93} INFO - 25/08/19 09:18:40 INFO MemoryStore: Block broadcast_4 stored as values in memory (estimated size 200.9 KiB, free 433.4 MiB)
[2025-08-19T09:18:40.090+0000] {subprocess.py:93} INFO - 25/08/19 09:18:40 INFO MemoryStore: Block broadcast_9 stored as values in memory (estimated size 200.9 KiB, free 433.2 MiB)
[2025-08-19T09:18:40.095+0000] {subprocess.py:93} INFO - 25/08/19 09:18:40 INFO MemoryStore: Block broadcast_1 stored as values in memory (estimated size 200.9 KiB, free 433.0 MiB)
[2025-08-19T09:18:40.126+0000] {subprocess.py:93} INFO - 25/08/19 09:18:40 INFO MemoryStore: Block broadcast_3 stored as values in memory (estimated size 200.9 KiB, free 432.4 MiB)
[2025-08-19T09:18:40.137+0000] {subprocess.py:93} INFO - 25/08/19 09:18:40 INFO MemoryStore: Block broadcast_2 stored as values in memory (estimated size 200.9 KiB, free 432.6 MiB)
[2025-08-19T09:18:40.141+0000] {subprocess.py:93} INFO - 25/08/19 09:18:40 INFO MemoryStore: Block broadcast_6 stored as values in memory (estimated size 200.9 KiB, free 432.4 MiB)
[2025-08-19T09:18:41.134+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO MemoryStore: Block broadcast_0_piece0 stored as bytes in memory (estimated size 34.8 KiB, free 432.3 MiB)
[2025-08-19T09:18:41.140+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO MemoryStore: Block broadcast_1_piece0 stored as bytes in memory (estimated size 34.8 KiB, free 432.2 MiB)
[2025-08-19T09:18:41.178+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO MemoryStore: Block broadcast_4_piece0 stored as bytes in memory (estimated size 34.8 KiB, free 432.2 MiB)
[2025-08-19T09:18:41.185+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO MemoryStore: Block broadcast_3_piece0 stored as bytes in memory (estimated size 34.8 KiB, free 432.1 MiB)
[2025-08-19T09:18:41.186+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO MemoryStore: Block broadcast_8_piece0 stored as bytes in memory (estimated size 34.8 KiB, free 432.3 MiB)
[2025-08-19T09:18:41.203+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO MemoryStore: Block broadcast_9_piece0 stored as bytes in memory (estimated size 34.8 KiB, free 432.3 MiB)
[2025-08-19T09:18:41.205+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO BlockManagerInfo: Added broadcast_0_piece0 in memory on b208641feb52:43727 (size: 34.8 KiB, free: 434.4 MiB)
[2025-08-19T09:18:41.214+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO MemoryStore: Block broadcast_6_piece0 stored as bytes in memory (estimated size 34.8 KiB, free 432.2 MiB)
[2025-08-19T09:18:41.224+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO MemoryStore: Block broadcast_7_piece0 stored as bytes in memory (estimated size 34.8 KiB, free 432.3 MiB)
[2025-08-19T09:18:41.225+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO BlockManagerInfo: Added broadcast_1_piece0 in memory on b208641feb52:43727 (size: 34.8 KiB, free: 434.3 MiB)
[2025-08-19T09:18:41.233+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO BlockManagerInfo: Added broadcast_4_piece0 in memory on b208641feb52:43727 (size: 34.8 KiB, free: 434.3 MiB)
[2025-08-19T09:18:41.241+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO MemoryStore: Block broadcast_5_piece0 stored as bytes in memory (estimated size 34.8 KiB, free 432.1 MiB)
[2025-08-19T09:18:41.252+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO SparkContext: Created broadcast 1 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:41.268+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO BlockManagerInfo: Added broadcast_3_piece0 in memory on b208641feb52:43727 (size: 34.8 KiB, free: 434.3 MiB)
[2025-08-19T09:18:41.286+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO SparkContext: Created broadcast 3 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:41.291+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO SparkContext: Created broadcast 0 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:41.295+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO BlockManagerInfo: Added broadcast_8_piece0 in memory on b208641feb52:43727 (size: 34.8 KiB, free: 434.2 MiB)
[2025-08-19T09:18:41.302+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO BlockManagerInfo: Added broadcast_9_piece0 in memory on b208641feb52:43727 (size: 34.8 KiB, free: 434.2 MiB)
[2025-08-19T09:18:41.306+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO BlockManagerInfo: Added broadcast_6_piece0 in memory on b208641feb52:43727 (size: 34.8 KiB, free: 434.2 MiB)
[2025-08-19T09:18:41.311+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO MemoryStore: Block broadcast_2_piece0 stored as bytes in memory (estimated size 34.8 KiB, free 432.1 MiB)
[2025-08-19T09:18:41.316+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO SparkContext: Created broadcast 4 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:41.329+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO SparkContext: Created broadcast 8 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:41.399+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO BlockManagerInfo: Added broadcast_7_piece0 in memory on b208641feb52:43727 (size: 34.8 KiB, free: 434.1 MiB)
[2025-08-19T09:18:41.435+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO SparkContext: Created broadcast 6 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:41.436+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO SparkContext: Created broadcast 9 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:41.444+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO SparkContext: Created broadcast 7 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:41.463+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO BlockManagerInfo: Added broadcast_5_piece0 in memory on b208641feb52:43727 (size: 34.8 KiB, free: 434.1 MiB)
[2025-08-19T09:18:41.477+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO BlockManagerInfo: Added broadcast_2_piece0 in memory on b208641feb52:43727 (size: 34.8 KiB, free: 434.1 MiB)
[2025-08-19T09:18:41.484+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:18:41.499+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:18:41.502+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167907 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:18:41.516+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:18:41.529+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:18:41.535+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO SparkContext: Created broadcast 5 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:41.535+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO FileSourceScanExec: Planning scan with bin packing, max size: 18875937 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:18:41.535+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:18:41.536+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:18:41.560+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:18:41.560+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO SparkContext: Created broadcast 2 from $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:41.691+0000] {subprocess.py:93} INFO - 25/08/19 09:18:41 INFO FileSourceScanExec: Planning scan with bin packing, max size: 25167895 bytes, open cost is considered as scanning 4194304 bytes.
[2025-08-19T09:18:43.269+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:43.314+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:43.463+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:43.532+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO DAGScheduler: Got job 0 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:18:43.534+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO DAGScheduler: Final stage: ResultStage 0 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:18:43.535+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:18:43.540+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:18:43.635+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO DAGScheduler: Submitting ResultStage 0 (MapPartitionsRDD[23] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:18:43.658+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:43.743+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:43.766+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:43.828+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:43.992+0000] {subprocess.py:93} INFO - 25/08/19 09:18:43 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:44.021+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:44.022+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO SparkContext: Starting job: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264
[2025-08-19T09:18:44.348+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO MemoryStore: Block broadcast_10 stored as values in memory (estimated size 14.1 KiB, free 432.1 MiB)
[2025-08-19T09:18:44.369+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO MemoryStore: Block broadcast_10_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.1 MiB)
[2025-08-19T09:18:44.389+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO BlockManagerInfo: Added broadcast_10_piece0 in memory on b208641feb52:43727 (size: 6.7 KiB, free: 434.1 MiB)
[2025-08-19T09:18:44.397+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO SparkContext: Created broadcast 10 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:18:44.532+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 0 (MapPartitionsRDD[23] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:18:44.541+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO TaskSchedulerImpl: Adding task set 0.0 with 2 tasks resource profile 0
[2025-08-19T09:18:44.660+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Got job 1 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:18:44.662+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Final stage: ResultStage 1 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:18:44.664+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:18:44.666+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:18:44.675+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Submitting ResultStage 1 (MapPartitionsRDD[24] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:18:44.680+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO MemoryStore: Block broadcast_11 stored as values in memory (estimated size 14.1 KiB, free 432.1 MiB)
[2025-08-19T09:18:44.683+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO MemoryStore: Block broadcast_11_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.1 MiB)
[2025-08-19T09:18:44.696+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO BlockManagerInfo: Added broadcast_11_piece0 in memory on b208641feb52:43727 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:18:44.721+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO SparkContext: Created broadcast 11 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:18:44.728+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 1 (MapPartitionsRDD[24] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:18:44.729+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO TaskSchedulerImpl: Adding task set 1.0 with 2 tasks resource profile 0
[2025-08-19T09:18:44.740+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Got job 2 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:18:44.741+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Final stage: ResultStage 2 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:18:44.746+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:18:44.754+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:18:44.781+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Submitting ResultStage 2 (MapPartitionsRDD[27] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:18:44.805+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO MemoryStore: Block broadcast_12 stored as values in memory (estimated size 14.1 KiB, free 432.0 MiB)
[2025-08-19T09:18:44.847+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO MemoryStore: Block broadcast_12_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.0 MiB)
[2025-08-19T09:18:44.853+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO BlockManagerInfo: Added broadcast_12_piece0 in memory on b208641feb52:43727 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:18:44.876+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO SparkContext: Created broadcast 12 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:18:44.876+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 2 (MapPartitionsRDD[27] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:18:44.877+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO TaskSchedulerImpl: Adding task set 2.0 with 2 tasks resource profile 0
[2025-08-19T09:18:44.889+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Got job 3 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:18:44.890+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Final stage: ResultStage 3 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:18:44.891+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:18:44.891+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:18:44.902+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Submitting ResultStage 3 (MapPartitionsRDD[32] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:18:44.940+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO MemoryStore: Block broadcast_13 stored as values in memory (estimated size 14.1 KiB, free 432.0 MiB)
[2025-08-19T09:18:44.949+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO MemoryStore: Block broadcast_13_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.0 MiB)
[2025-08-19T09:18:44.957+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO BlockManagerInfo: Added broadcast_13_piece0 in memory on b208641feb52:43727 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:18:44.973+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO SparkContext: Created broadcast 13 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:18:44.980+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 3 (MapPartitionsRDD[32] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:18:44.980+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO TaskSchedulerImpl: Adding task set 3.0 with 2 tasks resource profile 0
[2025-08-19T09:18:45.003+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Got job 4 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:18:45.003+0000] {subprocess.py:93} INFO - 25/08/19 09:18:44 INFO DAGScheduler: Final stage: ResultStage 4 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:18:45.003+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:18:45.011+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:18:45.011+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting ResultStage 4 (MapPartitionsRDD[37] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:18:45.066+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO MemoryStore: Block broadcast_14 stored as values in memory (estimated size 14.1 KiB, free 432.0 MiB)
[2025-08-19T09:18:45.073+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO MemoryStore: Block broadcast_14_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.0 MiB)
[2025-08-19T09:18:45.093+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO BlockManagerInfo: Added broadcast_14_piece0 in memory on b208641feb52:43727 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:18:45.114+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO SparkContext: Created broadcast 14 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:18:45.131+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 4 (MapPartitionsRDD[37] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:18:45.148+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO TaskSchedulerImpl: Adding task set 4.0 with 2 tasks resource profile 0
[2025-08-19T09:18:45.177+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Got job 5 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:18:45.198+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Final stage: ResultStage 5 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:18:45.201+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:18:45.202+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:18:45.202+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting ResultStage 5 (MapPartitionsRDD[34] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:18:45.264+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO MemoryStore: Block broadcast_15 stored as values in memory (estimated size 14.1 KiB, free 432.0 MiB)
[2025-08-19T09:18:45.296+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO MemoryStore: Block broadcast_15_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.0 MiB)
[2025-08-19T09:18:45.319+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO BlockManagerInfo: Added broadcast_15_piece0 in memory on b208641feb52:43727 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:18:45.365+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO SparkContext: Created broadcast 15 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:18:45.417+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 5 (MapPartitionsRDD[34] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:18:45.418+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO TaskSchedulerImpl: Adding task set 5.0 with 2 tasks resource profile 0
[2025-08-19T09:18:45.421+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Got job 6 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:18:45.440+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Final stage: ResultStage 6 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:18:45.461+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:18:45.461+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:18:45.461+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting ResultStage 6 (MapPartitionsRDD[39] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:18:45.462+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO MemoryStore: Block broadcast_16 stored as values in memory (estimated size 14.1 KiB, free 432.0 MiB)
[2025-08-19T09:18:45.528+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO MemoryStore: Block broadcast_16_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 432.0 MiB)
[2025-08-19T09:18:45.547+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO BlockManagerInfo: Added broadcast_16_piece0 in memory on b208641feb52:43727 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:18:45.570+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO SparkContext: Created broadcast 16 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:18:45.571+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 6 (MapPartitionsRDD[39] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:18:45.577+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO TaskSchedulerImpl: Adding task set 6.0 with 2 tasks resource profile 0
[2025-08-19T09:18:45.601+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Got job 7 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:18:45.602+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Final stage: ResultStage 7 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:18:45.602+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:18:45.603+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:18:45.615+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting ResultStage 7 (MapPartitionsRDD[35] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:18:45.645+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO MemoryStore: Block broadcast_17 stored as values in memory (estimated size 14.1 KiB, free 431.9 MiB)
[2025-08-19T09:18:45.670+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO MemoryStore: Block broadcast_17_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 431.9 MiB)
[2025-08-19T09:18:45.677+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO BlockManagerInfo: Added broadcast_17_piece0 in memory on b208641feb52:43727 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:18:45.683+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO SparkContext: Created broadcast 17 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:18:45.684+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 7 (MapPartitionsRDD[35] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:18:45.704+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO TaskSchedulerImpl: Adding task set 7.0 with 2 tasks resource profile 0
[2025-08-19T09:18:45.706+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Got job 8 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:18:45.712+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Final stage: ResultStage 8 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:18:45.715+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:18:45.740+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:18:45.745+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting ResultStage 8 (MapPartitionsRDD[30] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:18:45.805+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO MemoryStore: Block broadcast_18 stored as values in memory (estimated size 14.1 KiB, free 431.9 MiB)
[2025-08-19T09:18:45.831+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO MemoryStore: Block broadcast_18_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 431.9 MiB)
[2025-08-19T09:18:45.861+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO BlockManagerInfo: Added broadcast_18_piece0 in memory on b208641feb52:43727 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:18:45.878+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO SparkContext: Created broadcast 18 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:18:45.882+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 8 (MapPartitionsRDD[30] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:18:45.887+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO TaskSchedulerImpl: Adding task set 8.0 with 2 tasks resource profile 0
[2025-08-19T09:18:45.909+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Got job 9 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) with 2 output partitions
[2025-08-19T09:18:45.935+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Final stage: ResultStage 9 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264)
[2025-08-19T09:18:45.941+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Parents of final stage: List()
[2025-08-19T09:18:45.942+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Missing parents: List()
[2025-08-19T09:18:45.951+0000] {subprocess.py:93} INFO - 25/08/19 09:18:45 INFO DAGScheduler: Submitting ResultStage 9 (MapPartitionsRDD[29] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264), which has no missing parents
[2025-08-19T09:18:46.025+0000] {subprocess.py:93} INFO - 25/08/19 09:18:46 INFO MemoryStore: Block broadcast_19 stored as values in memory (estimated size 14.1 KiB, free 431.9 MiB)
[2025-08-19T09:18:46.042+0000] {subprocess.py:93} INFO - 25/08/19 09:18:46 INFO MemoryStore: Block broadcast_19_piece0 stored as bytes in memory (estimated size 6.7 KiB, free 431.9 MiB)
[2025-08-19T09:18:46.050+0000] {subprocess.py:93} INFO - 25/08/19 09:18:46 INFO BlockManagerInfo: Added broadcast_19_piece0 in memory on b208641feb52:43727 (size: 6.7 KiB, free: 434.0 MiB)
[2025-08-19T09:18:46.083+0000] {subprocess.py:93} INFO - 25/08/19 09:18:46 INFO SparkContext: Created broadcast 19 from broadcast at DAGScheduler.scala:1585
[2025-08-19T09:18:46.111+0000] {subprocess.py:93} INFO - 25/08/19 09:18:46 INFO DAGScheduler: Submitting 2 missing tasks from ResultStage 9 (MapPartitionsRDD[29] at $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) (first 15 tasks are for partitions Vector(0, 1))
[2025-08-19T09:18:46.112+0000] {subprocess.py:93} INFO - 25/08/19 09:18:46 INFO TaskSchedulerImpl: Adding task set 9.0 with 2 tasks resource profile 0
[2025-08-19T09:18:59.653+0000] {subprocess.py:93} INFO - 25/08/19 09:18:59 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:19:14.658+0000] {subprocess.py:93} INFO - 25/08/19 09:19:14 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:19:29.648+0000] {subprocess.py:93} INFO - 25/08/19 09:19:29 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:19:44.652+0000] {subprocess.py:93} INFO - 25/08/19 09:19:44 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:19:59.648+0000] {subprocess.py:93} INFO - 25/08/19 09:19:59 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:20:14.648+0000] {subprocess.py:93} INFO - 25/08/19 09:20:14 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:20:29.649+0000] {subprocess.py:93} INFO - 25/08/19 09:20:29 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:20:44.657+0000] {subprocess.py:93} INFO - 25/08/19 09:20:44 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:20:59.652+0000] {subprocess.py:93} INFO - 25/08/19 09:20:59 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:21:14.648+0000] {subprocess.py:93} INFO - 25/08/19 09:21:14 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:21:29.651+0000] {subprocess.py:93} INFO - 25/08/19 09:21:29 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:21:44.651+0000] {subprocess.py:93} INFO - 25/08/19 09:21:44 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:21:59.677+0000] {subprocess.py:93} INFO - 25/08/19 09:21:59 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:22:14.670+0000] {subprocess.py:93} INFO - 25/08/19 09:22:14 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:22:29.650+0000] {subprocess.py:93} INFO - 25/08/19 09:22:29 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:22:44.648+0000] {subprocess.py:93} INFO - 25/08/19 09:22:44 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:22:59.649+0000] {subprocess.py:93} INFO - 25/08/19 09:22:59 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:23:14.649+0000] {subprocess.py:93} INFO - 25/08/19 09:23:14 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:23:29.648+0000] {subprocess.py:93} INFO - 25/08/19 09:23:29 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:23:44.650+0000] {subprocess.py:93} INFO - 25/08/19 09:23:44 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:23:59.649+0000] {subprocess.py:93} INFO - 25/08/19 09:23:59 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:24:14.648+0000] {subprocess.py:93} INFO - 25/08/19 09:24:14 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:24:29.657+0000] {subprocess.py:93} INFO - 25/08/19 09:24:29 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:24:44.649+0000] {subprocess.py:93} INFO - 25/08/19 09:24:44 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:24:59.650+0000] {subprocess.py:93} INFO - 25/08/19 09:24:59 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:25:14.652+0000] {subprocess.py:93} INFO - 25/08/19 09:25:14 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:25:29.650+0000] {subprocess.py:93} INFO - 25/08/19 09:25:29 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:25:44.649+0000] {subprocess.py:93} INFO - 25/08/19 09:25:44 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:25:59.650+0000] {subprocess.py:93} INFO - 25/08/19 09:25:59 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:26:14.648+0000] {subprocess.py:93} INFO - 25/08/19 09:26:14 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:26:29.656+0000] {subprocess.py:93} INFO - 25/08/19 09:26:29 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:26:44.670+0000] {subprocess.py:93} INFO - 25/08/19 09:26:44 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:26:59.680+0000] {subprocess.py:93} INFO - 25/08/19 09:26:59 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:27:14.653+0000] {subprocess.py:93} INFO - 25/08/19 09:27:14 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:27:29.648+0000] {subprocess.py:93} INFO - 25/08/19 09:27:29 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:27:44.656+0000] {subprocess.py:93} INFO - 25/08/19 09:27:44 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:27:59.647+0000] {subprocess.py:93} INFO - 25/08/19 09:27:59 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:28:04.075+0000] {subprocess.py:93} INFO - 25/08/19 09:28:04 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20250819091746-0006/0 on worker-20250819090904-172.18.0.12-34859 (172.18.0.12:34859) with 4 core(s)
[2025-08-19T09:28:04.090+0000] {subprocess.py:93} INFO - 25/08/19 09:28:04 INFO StandaloneSchedulerBackend: Granted executor ID app-20250819091746-0006/0 on hostPort 172.18.0.12:34859 with 4 core(s), 1024.0 MiB RAM
[2025-08-19T09:28:04.098+0000] {subprocess.py:93} INFO - 25/08/19 09:28:04 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20250819091746-0006/1 on worker-20250819090904-172.18.0.17-46791 (172.18.0.17:46791) with 4 core(s)
[2025-08-19T09:28:04.102+0000] {subprocess.py:93} INFO - 25/08/19 09:28:04 INFO StandaloneSchedulerBackend: Granted executor ID app-20250819091746-0006/1 on hostPort 172.18.0.17:46791 with 4 core(s), 1024.0 MiB RAM
[2025-08-19T09:28:04.128+0000] {subprocess.py:93} INFO - 25/08/19 09:28:04 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20250819091746-0006/2 on worker-20250819090906-172.18.0.16-36117 (172.18.0.16:36117) with 4 core(s)
[2025-08-19T09:28:04.132+0000] {subprocess.py:93} INFO - 25/08/19 09:28:04 INFO StandaloneSchedulerBackend: Granted executor ID app-20250819091746-0006/2 on hostPort 172.18.0.16:36117 with 4 core(s), 1024.0 MiB RAM
[2025-08-19T09:28:04.624+0000] {subprocess.py:93} INFO - 25/08/19 09:28:04 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250819091746-0006/1 is now RUNNING
[2025-08-19T09:28:04.716+0000] {subprocess.py:93} INFO - 25/08/19 09:28:04 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250819091746-0006/0 is now RUNNING
[2025-08-19T09:28:04.864+0000] {subprocess.py:93} INFO - 25/08/19 09:28:04 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250819091746-0006/2 is now RUNNING
[2025-08-19T09:28:14.651+0000] {subprocess.py:93} INFO - 25/08/19 09:28:14 WARN TaskSchedulerImpl: Initial job has not accepted any resources; check your cluster UI to ensure that workers are registered and have sufficient resources
[2025-08-19T09:28:19.278+0000] {subprocess.py:93} INFO - 25/08/19 09:28:19 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.18.0.17:60112) with ID 1,  ResourceProfileId 0
[2025-08-19T09:28:19.753+0000] {subprocess.py:93} INFO - 25/08/19 09:28:19 INFO BlockManagerMasterEndpoint: Registering block manager 172.18.0.17:39895 with 434.4 MiB RAM, BlockManagerId(1, 172.18.0.17, 39895, None)
[2025-08-19T09:28:20.552+0000] {subprocess.py:93} INFO - 25/08/19 09:28:20 INFO TaskSetManager: Starting task 0.0 in stage 0.0 (TID 0) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:20.561+0000] {subprocess.py:93} INFO - 25/08/19 09:28:20 INFO TaskSetManager: Starting task 1.0 in stage 0.0 (TID 1) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:20.564+0000] {subprocess.py:93} INFO - 25/08/19 09:28:20 INFO TaskSetManager: Starting task 0.0 in stage 1.0 (TID 2) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:20.566+0000] {subprocess.py:93} INFO - 25/08/19 09:28:20 INFO TaskSetManager: Starting task 1.0 in stage 1.0 (TID 3) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:20.578+0000] {subprocess.py:93} INFO - 25/08/19 09:28:20 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.18.0.16:45096) with ID 2,  ResourceProfileId 0
[2025-08-19T09:28:20.853+0000] {subprocess.py:93} INFO - 25/08/19 09:28:20 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.18.0.12:54780) with ID 0,  ResourceProfileId 0
[2025-08-19T09:28:21.049+0000] {subprocess.py:93} INFO - 25/08/19 09:28:21 INFO BlockManagerMasterEndpoint: Registering block manager 172.18.0.12:44171 with 434.4 MiB RAM, BlockManagerId(0, 172.18.0.12, 44171, None)
[2025-08-19T09:28:21.284+0000] {subprocess.py:93} INFO - 25/08/19 09:28:21 INFO BlockManagerMasterEndpoint: Registering block manager 172.18.0.16:39187 with 434.4 MiB RAM, BlockManagerId(2, 172.18.0.16, 39187, None)
[2025-08-19T09:28:21.794+0000] {subprocess.py:93} INFO - 25/08/19 09:28:21 INFO TaskSetManager: Starting task 0.0 in stage 2.0 (TID 4) (172.18.0.12, executor 0, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:21.795+0000] {subprocess.py:93} INFO - 25/08/19 09:28:21 INFO TaskSetManager: Starting task 1.0 in stage 2.0 (TID 5) (172.18.0.12, executor 0, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:21.814+0000] {subprocess.py:93} INFO - 25/08/19 09:28:21 INFO TaskSetManager: Starting task 0.0 in stage 3.0 (TID 6) (172.18.0.12, executor 0, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:21.817+0000] {subprocess.py:93} INFO - 25/08/19 09:28:21 INFO TaskSetManager: Starting task 1.0 in stage 3.0 (TID 7) (172.18.0.12, executor 0, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:22.043+0000] {subprocess.py:93} INFO - 25/08/19 09:28:22 INFO TaskSetManager: Starting task 0.0 in stage 4.0 (TID 8) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:22.044+0000] {subprocess.py:93} INFO - 25/08/19 09:28:22 INFO TaskSetManager: Starting task 1.0 in stage 4.0 (TID 9) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:22.046+0000] {subprocess.py:93} INFO - 25/08/19 09:28:22 INFO TaskSetManager: Starting task 0.0 in stage 5.0 (TID 10) (172.18.0.16, executor 2, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:22.051+0000] {subprocess.py:93} INFO - 25/08/19 09:28:22 INFO TaskSetManager: Starting task 1.0 in stage 5.0 (TID 11) (172.18.0.16, executor 2, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:22.108+0000] {subprocess.py:93} INFO - 25/08/19 09:28:22 INFO BlockManagerInfo: Added broadcast_10_piece0 in memory on 172.18.0.17:39895 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:28:22.112+0000] {subprocess.py:93} INFO - 25/08/19 09:28:22 INFO BlockManagerInfo: Added broadcast_11_piece0 in memory on 172.18.0.17:39895 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:28:23.114+0000] {subprocess.py:93} INFO - 25/08/19 09:28:23 INFO BlockManagerInfo: Added broadcast_13_piece0 in memory on 172.18.0.12:44171 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:28:23.122+0000] {subprocess.py:93} INFO - 25/08/19 09:28:23 INFO BlockManagerInfo: Added broadcast_12_piece0 in memory on 172.18.0.12:44171 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:28:23.535+0000] {subprocess.py:93} INFO - 25/08/19 09:28:23 INFO BlockManagerInfo: Added broadcast_14_piece0 in memory on 172.18.0.16:39187 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:28:23.550+0000] {subprocess.py:93} INFO - 25/08/19 09:28:23 INFO BlockManagerInfo: Added broadcast_15_piece0 in memory on 172.18.0.16:39187 (size: 6.7 KiB, free: 434.4 MiB)
[2025-08-19T09:28:26.417+0000] {subprocess.py:93} INFO - 25/08/19 09:28:26 INFO BlockManagerInfo: Added broadcast_4_piece0 in memory on 172.18.0.17:39895 (size: 34.8 KiB, free: 434.4 MiB)
[2025-08-19T09:28:26.436+0000] {subprocess.py:93} INFO - 25/08/19 09:28:26 INFO BlockManagerInfo: Added broadcast_2_piece0 in memory on 172.18.0.17:39895 (size: 34.8 KiB, free: 434.3 MiB)
[2025-08-19T09:28:27.673+0000] {subprocess.py:93} INFO - 25/08/19 09:28:27 INFO BlockManagerInfo: Added broadcast_9_piece0 in memory on 172.18.0.16:39187 (size: 34.8 KiB, free: 434.4 MiB)
[2025-08-19T09:28:27.682+0000] {subprocess.py:93} INFO - 25/08/19 09:28:27 INFO BlockManagerInfo: Added broadcast_5_piece0 in memory on 172.18.0.16:39187 (size: 34.8 KiB, free: 434.3 MiB)
[2025-08-19T09:28:28.174+0000] {subprocess.py:93} INFO - 25/08/19 09:28:28 INFO BlockManagerInfo: Added broadcast_7_piece0 in memory on 172.18.0.12:44171 (size: 34.8 KiB, free: 434.4 MiB)
[2025-08-19T09:28:28.176+0000] {subprocess.py:93} INFO - 25/08/19 09:28:28 INFO BlockManagerInfo: Added broadcast_6_piece0 in memory on 172.18.0.12:44171 (size: 34.8 KiB, free: 434.3 MiB)
[2025-08-19T09:28:30.438+0000] {subprocess.py:93} INFO - 25/08/19 09:28:30 INFO TaskSetManager: Starting task 0.0 in stage 6.0 (TID 12) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:30.480+0000] {subprocess.py:93} INFO - 25/08/19 09:28:30 INFO TaskSetManager: Starting task 1.0 in stage 6.0 (TID 13) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:30.515+0000] {subprocess.py:93} INFO - 25/08/19 09:28:30 INFO TaskSetManager: Starting task 0.0 in stage 7.0 (TID 14) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:30.555+0000] {subprocess.py:93} INFO - 25/08/19 09:28:30 INFO TaskSetManager: Starting task 1.0 in stage 7.0 (TID 15) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:30.629+0000] {subprocess.py:93} INFO - 25/08/19 09:28:30 WARN TaskSetManager: Lost task 1.0 in stage 0.0 (TID 1) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00011-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:30.631+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:30.633+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:30.634+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:30.637+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:30.643+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:30.647+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:30.649+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-08-19T09:28:30.654+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:30.656+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:30.660+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:30.664+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:30.666+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:30.668+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:30.671+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:30.674+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:30.678+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:30.681+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:30.684+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:30.685+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:30.685+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:30.686+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:30.686+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:30.686+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:30.686+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:30.702+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:30.703+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:30.703+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:30.712+0000] {subprocess.py:93} INFO - 25/08/19 09:28:30 WARN TaskSetManager: Lost task 1.0 in stage 1.0 (TID 3) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00011-be9e9d80-b132-4d09-8a6f-78394356cb1f-c000.avro
[2025-08-19T09:28:30.713+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:30.718+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:30.723+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:30.726+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:30.729+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:30.732+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:30.738+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage3.processNext(Unknown Source)
[2025-08-19T09:28:30.740+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:30.743+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:30.746+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:30.749+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:30.751+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:30.752+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:30.754+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:30.757+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:30.766+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:30.769+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:30.772+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:30.775+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:30.776+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:30.777+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:30.777+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:30.778+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:30.778+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:30.778+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:30.779+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:30.779+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:30.779+0000] {subprocess.py:93} INFO - 25/08/19 09:28:30 WARN TaskSetManager: Lost task 0.0 in stage 0.0 (TID 0) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00005-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:30.780+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:30.780+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:30.780+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:30.780+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:30.781+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:30.781+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:30.781+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-08-19T09:28:30.781+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:30.782+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:30.782+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:30.782+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:30.782+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:30.783+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:30.783+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:30.783+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:30.784+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:30.784+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:30.784+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:30.785+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:30.785+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:30.785+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:30.785+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:30.786+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:30.786+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:30.786+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:30.787+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:30.787+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:30.788+0000] {subprocess.py:93} INFO - 25/08/19 09:28:30 WARN TaskSetManager: Lost task 0.0 in stage 1.0 (TID 2) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00005-be9e9d80-b132-4d09-8a6f-78394356cb1f-c000.avro
[2025-08-19T09:28:30.788+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:30.788+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:30.788+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:30.789+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:30.789+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:30.789+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:30.789+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage3.processNext(Unknown Source)
[2025-08-19T09:28:30.790+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:30.790+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:30.790+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:30.791+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:30.791+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:30.791+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:30.791+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:30.792+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:30.792+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:30.792+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:30.792+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:30.793+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:30.793+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:30.793+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:30.794+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:30.794+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:30.794+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:30.794+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:30.795+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:30.795+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:30.898+0000] {subprocess.py:93} INFO - 25/08/19 09:28:30 INFO BlockManagerInfo: Added broadcast_16_piece0 in memory on 172.18.0.17:39895 (size: 6.7 KiB, free: 434.3 MiB)
[2025-08-19T09:28:30.910+0000] {subprocess.py:93} INFO - 25/08/19 09:28:30 INFO BlockManagerInfo: Added broadcast_17_piece0 in memory on 172.18.0.17:39895 (size: 6.7 KiB, free: 434.3 MiB)
[2025-08-19T09:28:31.099+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO BlockManagerInfo: Added broadcast_1_piece0 in memory on 172.18.0.17:39895 (size: 34.8 KiB, free: 434.3 MiB)
[2025-08-19T09:28:31.107+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO BlockManagerInfo: Added broadcast_0_piece0 in memory on 172.18.0.17:39895 (size: 34.8 KiB, free: 434.2 MiB)
[2025-08-19T09:28:31.293+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 0.1 in stage 0.0 (TID 16) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:31.294+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 WARN TaskSetManager: Lost task 0.0 in stage 6.0 (TID 12) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00005-61492aaa-0164-4bc3-bd73-36af6b214e44-c000.avro
[2025-08-19T09:28:31.295+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:31.296+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:31.297+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:31.298+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:31.299+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:31.301+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:31.303+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage8.processNext(Unknown Source)
[2025-08-19T09:28:31.304+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:31.306+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:31.307+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:31.310+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:31.311+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:31.312+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:31.313+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:31.315+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:31.316+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:31.316+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:31.318+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:31.321+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:31.322+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:31.323+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:31.325+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:31.326+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:31.327+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:31.328+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:31.331+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:31.332+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:31.333+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 1.1 in stage 0.0 (TID 17) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:31.334+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 WARN TaskSetManager: Lost task 1.0 in stage 6.0 (TID 13) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00011-61492aaa-0164-4bc3-bd73-36af6b214e44-c000.avro
[2025-08-19T09:28:31.335+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:31.337+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:31.338+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:31.339+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:31.341+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:31.342+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:31.345+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage8.processNext(Unknown Source)
[2025-08-19T09:28:31.346+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:31.347+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:31.348+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:31.349+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:31.350+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:31.351+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:31.353+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:31.354+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:31.356+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:31.357+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:31.358+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:31.359+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:31.360+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:31.361+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:31.362+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:31.363+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:31.365+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:31.366+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:31.366+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:31.367+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:31.367+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 0.1 in stage 1.0 (TID 18) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:31.367+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 WARN TaskSetManager: Lost task 1.0 in stage 7.0 (TID 15) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/DE_category_id/part-00011-c990aed5-9084-442f-b92a-e32baa2e7dcb-c000.avro
[2025-08-19T09:28:31.368+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:31.369+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:31.370+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:31.371+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:31.372+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:31.373+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:31.374+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage2.processNext(Unknown Source)
[2025-08-19T09:28:31.375+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:31.376+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:31.377+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:31.378+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:31.379+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:31.380+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:31.381+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:31.382+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:31.383+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:31.384+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:31.385+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:31.385+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:31.386+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:31.387+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:31.388+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:31.389+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:31.391+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:31.391+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:31.392+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:31.393+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:31.394+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 1.1 in stage 1.0 (TID 19) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:31.395+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 WARN TaskSetManager: Lost task 0.0 in stage 7.0 (TID 14) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/DE_category_id/part-00005-c990aed5-9084-442f-b92a-e32baa2e7dcb-c000.avro
[2025-08-19T09:28:31.396+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:31.398+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:31.399+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:31.400+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:31.401+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:31.402+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:31.403+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage2.processNext(Unknown Source)
[2025-08-19T09:28:31.404+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:31.404+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:31.405+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:31.406+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:31.407+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:31.408+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:31.409+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:31.410+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:31.413+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:31.414+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:31.415+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:31.416+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:31.417+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:31.417+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:31.418+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:31.418+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:31.419+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:31.423+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:31.424+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:31.425+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:31.492+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 1.1 in stage 6.0 (TID 20) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:31.507+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 1.1 in stage 0.0 (TID 17) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00011-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:31.508+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:28:31.511+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 1.2 in stage 0.0 (TID 21) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:31.522+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 0.1 in stage 0.0 (TID 16) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00005-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:31.523+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:28:31.577+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 0.2 in stage 0.0 (TID 22) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:31.590+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 0.1 in stage 1.0 (TID 18) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00005-be9e9d80-b132-4d09-8a6f-78394356cb1f-c000.avro
[2025-08-19T09:28:31.591+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:28:31.624+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 0.2 in stage 1.0 (TID 23) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:31.632+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 1.1 in stage 1.0 (TID 19) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00011-be9e9d80-b132-4d09-8a6f-78394356cb1f-c000.avro
[2025-08-19T09:28:31.633+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:28:31.664+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 1.2 in stage 1.0 (TID 24) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:31.668+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 1.1 in stage 6.0 (TID 20) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00011-61492aaa-0164-4bc3-bd73-36af6b214e44-c000.avro
[2025-08-19T09:28:31.669+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:28:31.707+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 1.2 in stage 6.0 (TID 25) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:31.719+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 1.2 in stage 0.0 (TID 21) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00011-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:31.720+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2025-08-19T09:28:31.721+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 1.3 in stage 0.0 (TID 26) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:31.735+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 0.2 in stage 0.0 (TID 22) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00005-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:31.737+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2025-08-19T09:28:31.826+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 0.3 in stage 0.0 (TID 27) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:31.831+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 0.2 in stage 1.0 (TID 23) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00005-be9e9d80-b132-4d09-8a6f-78394356cb1f-c000.avro
[2025-08-19T09:28:31.832+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2025-08-19T09:28:31.853+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 0.3 in stage 1.0 (TID 28) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:31.867+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 1.2 in stage 1.0 (TID 24) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00011-be9e9d80-b132-4d09-8a6f-78394356cb1f-c000.avro
[2025-08-19T09:28:31.869+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2025-08-19T09:28:31.930+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 1.3 in stage 1.0 (TID 29) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:31.957+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 1.2 in stage 6.0 (TID 25) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00011-61492aaa-0164-4bc3-bd73-36af6b214e44-c000.avro
[2025-08-19T09:28:31.958+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2025-08-19T09:28:31.961+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Starting task 1.3 in stage 6.0 (TID 30) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:31.963+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSetManager: Lost task 1.3 in stage 0.0 (TID 26) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00011-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:31.971+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2025-08-19T09:28:31.973+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 ERROR TaskSetManager: Task 1 in stage 0.0 failed 4 times; aborting job
[2025-08-19T09:28:32.005+0000] {subprocess.py:93} INFO - 25/08/19 09:28:31 INFO TaskSchedulerImpl: Cancelling stage 0
[2025-08-19T09:28:32.008+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Killing all running tasks in stage 0: Stage cancelled: Job aborted due to stage failure: Task 1 in stage 0.0 failed 4 times, most recent failure: Lost task 1.3 in stage 0.0 (TID 26) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00011-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:32.010+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:32.014+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:32.017+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:32.020+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:32.026+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:32.029+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:32.033+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-08-19T09:28:32.034+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:32.035+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:32.039+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:32.039+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:32.042+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:32.042+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:32.042+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:32.043+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:32.046+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:32.047+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:32.049+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:32.050+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:32.050+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:32.050+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:32.051+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:32.055+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:32.056+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:32.057+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:32.061+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:32.065+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:32.066+0000] {subprocess.py:93} INFO - Driver stacktrace:
[2025-08-19T09:28:32.067+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Stage 0 was cancelled
[2025-08-19T09:28:32.076+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSetManager: Starting task 0.1 in stage 6.0 (TID 31) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:32.080+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: ResultStage 0 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 587.842 s due to Job aborted due to stage failure: Task 1 in stage 0.0 failed 4 times, most recent failure: Lost task 1.3 in stage 0.0 (TID 26) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00011-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:32.082+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:32.089+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:32.091+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:32.092+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:32.093+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:32.099+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:32.103+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-08-19T09:28:32.107+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:32.108+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:32.112+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:32.112+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:32.115+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:32.118+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:32.120+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:32.124+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:32.126+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:32.126+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:32.131+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:32.133+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:32.137+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:32.139+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:32.141+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:32.142+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:32.142+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:32.143+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:32.151+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:32.152+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:32.152+0000] {subprocess.py:93} INFO - Driver stacktrace:
[2025-08-19T09:28:32.152+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSetManager: Lost task 0.3 in stage 0.0 (TID 27) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00005-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:32.156+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2025-08-19T09:28:32.159+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 0.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.171+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSetManager: Starting task 0.1 in stage 7.0 (TID 32) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:32.213+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSetManager: Lost task 1.3 in stage 1.0 (TID 29) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00011-be9e9d80-b132-4d09-8a6f-78394356cb1f-c000.avro
[2025-08-19T09:28:32.215+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2025-08-19T09:28:32.229+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 ERROR TaskSetManager: Task 1 in stage 1.0 failed 4 times; aborting job
[2025-08-19T09:28:32.241+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Job 0 failed: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264, took 588.973644 s
[2025-08-19T09:28:32.264+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Cancelling stage 1
[2025-08-19T09:28:32.265+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Killing all running tasks in stage 1: Stage cancelled: Job aborted due to stage failure: Task 1 in stage 1.0 failed 4 times, most recent failure: Lost task 1.3 in stage 1.0 (TID 29) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00011-be9e9d80-b132-4d09-8a6f-78394356cb1f-c000.avro
[2025-08-19T09:28:32.271+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:32.275+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:32.282+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:32.288+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:32.290+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:32.299+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:32.300+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage3.processNext(Unknown Source)
[2025-08-19T09:28:32.302+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:32.303+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:32.303+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:32.317+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:32.319+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:32.329+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:32.332+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:32.341+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:32.346+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:32.348+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:32.348+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:32.351+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:32.354+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:32.359+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:32.359+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:32.373+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:32.382+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:32.386+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:32.386+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:32.406+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:32.407+0000] {subprocess.py:93} INFO - Driver stacktrace:
[2025-08-19T09:28:32.410+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Stage 1 was cancelled
[2025-08-19T09:28:32.415+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: ResultStage 1 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 587.620 s due to Job aborted due to stage failure: Task 1 in stage 1.0 failed 4 times, most recent failure: Lost task 1.3 in stage 1.0 (TID 29) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00011-be9e9d80-b132-4d09-8a6f-78394356cb1f-c000.avro
[2025-08-19T09:28:32.416+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:32.421+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:32.423+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:32.426+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:32.442+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:32.445+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:32.446+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage3.processNext(Unknown Source)
[2025-08-19T09:28:32.446+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:32.451+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:32.459+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:32.460+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:32.460+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:32.460+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:32.461+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:32.467+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:32.469+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:32.470+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:32.470+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:32.479+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:32.480+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:32.481+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:32.484+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:32.487+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:32.490+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:32.492+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:32.496+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:32.502+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:32.503+0000] {subprocess.py:93} INFO - Driver stacktrace:
[2025-08-19T09:28:32.503+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Job 1 failed: $anonfun$withThreadLocalCaptured$1 at FutureTask.java:264, took 588.959280 s
[2025-08-19T09:28:32.504+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 05592cb8-5a33-4d94-8b5b-4e3cc33068db)
[2025-08-19T09:28:32.504+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 3887a063-01d8-4cee-9056-9c7190f25286)
[2025-08-19T09:28:32.504+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 9dc6c13b-6d30-43bc-8fa5-57b878f40abb)
[2025-08-19T09:28:32.512+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 1500a951-a34b-4157-a998-3dee31bb4d7e)
[2025-08-19T09:28:32.516+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId f2692b15-a478-4826-8762-da64c74c618a)
[2025-08-19T09:28:32.520+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 2f09d15a-236e-44c1-9a90-eb305f3a4925)
[2025-08-19T09:28:32.520+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 6db28d96-ea1b-4f7c-977a-7348e8de2961)
[2025-08-19T09:28:32.527+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId bb5cda2a-bf1c-442e-b446-5248757c332d)
[2025-08-19T09:28:32.532+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: Asked to cancel jobs with tag broadcast exchange (runId 56c3bfb1-92c2-47af-aaff-6113d5d0d318)
[2025-08-19T09:28:32.534+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSetManager: Starting task 1.1 in stage 7.0 (TID 33) (172.18.0.17, executor 1, partition 1, ANY, 9620 bytes)
[2025-08-19T09:28:32.537+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSetManager: Lost task 0.3 in stage 1.0 (TID 28) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/FR_category_id/part-00005-be9e9d80-b132-4d09-8a6f-78394356cb1f-c000.avro
[2025-08-19T09:28:32.539+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2025-08-19T09:28:32.541+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 1.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.544+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSetManager: Starting task 0.0 in stage 8.0 (TID 34) (172.18.0.17, executor 1, partition 0, ANY, 9620 bytes)
[2025-08-19T09:28:32.546+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSetManager: Lost task 1.3 in stage 6.0 (TID 30) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00011-61492aaa-0164-4bc3-bd73-36af6b214e44-c000.avro
[2025-08-19T09:28:32.550+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2025-08-19T09:28:32.552+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 ERROR TaskSetManager: Task 1 in stage 6.0 failed 4 times; aborting job
[2025-08-19T09:28:32.554+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Cancelling stage 8
[2025-08-19T09:28:32.558+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Killing all running tasks in stage 8: Stage cancelled: Job 8 cancelled part of cancelled job tag broadcast exchange (runId 05592cb8-5a33-4d94-8b5b-4e3cc33068db)
[2025-08-19T09:28:32.561+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Stage 8 was cancelled
[2025-08-19T09:28:32.561+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: ResultStage 8 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 586.748 s due to Job 8 cancelled part of cancelled job tag broadcast exchange (runId 05592cb8-5a33-4d94-8b5b-4e3cc33068db)
[2025-08-19T09:28:32.562+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Cancelling stage 7
[2025-08-19T09:28:32.562+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Killing all running tasks in stage 7: Stage cancelled: Job 7 cancelled part of cancelled job tag broadcast exchange (runId 3887a063-01d8-4cee-9056-9c7190f25286)
[2025-08-19T09:28:32.562+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Stage 7 was cancelled
[2025-08-19T09:28:32.563+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: ResultStage 7 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 586.873 s due to Job 7 cancelled part of cancelled job tag broadcast exchange (runId 3887a063-01d8-4cee-9056-9c7190f25286)
[2025-08-19T09:28:32.570+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Cancelling stage 2
[2025-08-19T09:28:32.573+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Killing all running tasks in stage 2: Stage cancelled: Job 2 cancelled part of cancelled job tag broadcast exchange (runId 1500a951-a34b-4157-a998-3dee31bb4d7e)
[2025-08-19T09:28:32.576+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Stage 2 was cancelled
[2025-08-19T09:28:32.584+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: ResultStage 2 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 587.746 s due to Job 2 cancelled part of cancelled job tag broadcast exchange (runId 1500a951-a34b-4157-a998-3dee31bb4d7e)
[2025-08-19T09:28:32.587+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Cancelling stage 4
[2025-08-19T09:28:32.594+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Killing all running tasks in stage 4: Stage cancelled: Job 4 cancelled part of cancelled job tag broadcast exchange (runId f2692b15-a478-4826-8762-da64c74c618a)
[2025-08-19T09:28:32.599+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Stage 4 was cancelled
[2025-08-19T09:28:32.600+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: ResultStage 4 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 587.544 s due to Job 4 cancelled part of cancelled job tag broadcast exchange (runId f2692b15-a478-4826-8762-da64c74c618a)
[2025-08-19T09:28:32.608+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Cancelling stage 9
[2025-08-19T09:28:32.615+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Killing all running tasks in stage 9: Stage cancelled: Job 9 cancelled part of cancelled job tag broadcast exchange (runId 2f09d15a-236e-44c1-9a90-eb305f3a4925)
[2025-08-19T09:28:32.615+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 9.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.616+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Stage 9 was cancelled
[2025-08-19T09:28:32.616+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: ResultStage 9 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 586.639 s due to Job 9 cancelled part of cancelled job tag broadcast exchange (runId 2f09d15a-236e-44c1-9a90-eb305f3a4925)
[2025-08-19T09:28:32.645+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Cancelling stage 6
[2025-08-19T09:28:32.647+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Killing all running tasks in stage 6: Stage cancelled: Job 6 cancelled part of cancelled job tag broadcast exchange (runId 6db28d96-ea1b-4f7c-977a-7348e8de2961)
[2025-08-19T09:28:32.650+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 6.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.658+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Stage 6 was cancelled
[2025-08-19T09:28:32.666+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSetManager: Lost task 0.1 in stage 6.0 (TID 31) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/MX_category_id/part-00005-61492aaa-0164-4bc3-bd73-36af6b214e44-c000.avro
[2025-08-19T09:28:32.667+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:28:32.669+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 6.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.676+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: ResultStage 6 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 587.249 s due to Job 6 cancelled part of cancelled job tag broadcast exchange (runId 6db28d96-ea1b-4f7c-977a-7348e8de2961)
[2025-08-19T09:28:32.686+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Cancelling stage 5
[2025-08-19T09:28:32.690+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Killing all running tasks in stage 5: Stage cancelled: Job 5 cancelled part of cancelled job tag broadcast exchange (runId bb5cda2a-bf1c-442e-b446-5248757c332d)
[2025-08-19T09:28:32.692+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Stage 5 was cancelled
[2025-08-19T09:28:32.695+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: ResultStage 5 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 587.450 s due to Job 5 cancelled part of cancelled job tag broadcast exchange (runId bb5cda2a-bf1c-442e-b446-5248757c332d)
[2025-08-19T09:28:32.701+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Cancelling stage 3
[2025-08-19T09:28:32.704+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Killing all running tasks in stage 3: Stage cancelled: Job 3 cancelled part of cancelled job tag broadcast exchange (runId 56c3bfb1-92c2-47af-aaff-6113d5d0d318)
[2025-08-19T09:28:32.712+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Stage 3 was cancelled
[2025-08-19T09:28:32.728+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO DAGScheduler: ResultStage 3 ($anonfun$withThreadLocalCaptured$1 at FutureTask.java:264) failed in 587.821 s due to Job 3 cancelled part of cancelled job tag broadcast exchange (runId 56c3bfb1-92c2-47af-aaff-6113d5d0d318)
[2025-08-19T09:28:32.761+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSetManager: Lost task 0.1 in stage 7.0 (TID 32) on 172.18.0.17, executor 1: org.apache.spark.SparkFileNotFoundException (File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/DE_category_id/part-00005-c990aed5-9084-442f-b92a-e32baa2e7dcb-c000.avro
[2025-08-19T09:28:32.762+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2025-08-19T09:28:32.765+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 7.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.767+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 WARN TaskSetManager: Lost task 0.0 in stage 8.0 (TID 34) (172.18.0.17 executor 1): TaskKilled (Stage cancelled: Job 8 cancelled part of cancelled job tag broadcast exchange (runId 05592cb8-5a33-4d94-8b5b-4e3cc33068db))
[2025-08-19T09:28:32.792+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 8.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.796+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 WARN TaskSetManager: Lost task 1.1 in stage 7.0 (TID 33) (172.18.0.17 executor 1): TaskKilled (Stage cancelled: Job 7 cancelled part of cancelled job tag broadcast exchange (runId 3887a063-01d8-4cee-9056-9c7190f25286))
[2025-08-19T09:28:32.798+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 7.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.802+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 WARN TaskSetManager: Lost task 0.0 in stage 4.0 (TID 8) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/IN_category_id/part-00005-cde2eb43-4094-49e1-81d3-bea69c6173d5-c000.avro
[2025-08-19T09:28:32.805+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:32.806+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:32.809+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:32.810+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:32.810+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:32.815+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:32.817+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage5.processNext(Unknown Source)
[2025-08-19T09:28:32.821+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:32.822+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:32.823+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:32.823+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:32.823+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:32.827+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:32.828+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:32.828+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:32.829+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:32.838+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:32.840+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:32.840+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:32.841+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:32.842+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:32.843+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:32.845+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:32.845+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:32.845+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:32.849+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:32.851+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:32.854+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 4.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.856+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 WARN TaskSetManager: Lost task 0.0 in stage 5.0 (TID 10) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/RU_category_id/part-00005-30f98837-2089-4970-ae3a-09b6ee0301bb-c000.avro
[2025-08-19T09:28:32.857+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:32.861+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:32.863+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:32.864+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:32.864+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:32.865+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:32.869+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage9.processNext(Unknown Source)
[2025-08-19T09:28:32.870+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:32.871+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:32.873+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:32.875+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:32.876+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:32.876+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:32.876+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:32.877+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:32.877+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:32.877+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:32.878+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:32.885+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:32.886+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:32.886+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:32.887+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:32.895+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:32.898+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:32.899+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:32.899+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:32.900+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:32.905+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 5.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.906+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 WARN TaskSetManager: Lost task 1.0 in stage 5.0 (TID 11) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/RU_category_id/part-00011-30f98837-2089-4970-ae3a-09b6ee0301bb-c000.avro
[2025-08-19T09:28:32.911+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:32.911+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:32.912+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:32.921+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:32.923+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:32.924+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:32.925+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage9.processNext(Unknown Source)
[2025-08-19T09:28:32.927+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:32.931+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:32.934+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:32.938+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:32.939+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:32.940+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:32.941+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:32.941+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:32.942+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:32.943+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:32.949+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:32.950+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:32.954+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:32.955+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:32.956+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:32.957+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:32.958+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:32.959+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:32.960+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:32.961+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:32.962+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 5.0, whose tasks have all completed, from pool
[2025-08-19T09:28:32.963+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 WARN TaskSetManager: Lost task 1.0 in stage 4.0 (TID 9) (172.18.0.16 executor 2): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/IN_category_id/part-00011-cde2eb43-4094-49e1-81d3-bea69c6173d5-c000.avro
[2025-08-19T09:28:32.963+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:32.965+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:32.965+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:32.966+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:32.967+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:32.968+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:32.969+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage5.processNext(Unknown Source)
[2025-08-19T09:28:32.970+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:32.971+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:32.972+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:32.973+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:32.974+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:32.975+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:32.976+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:32.977+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:32.978+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:32.979+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:32.980+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:32.981+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:32.982+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:32.982+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:32.984+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:32.985+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:32.986+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:32.986+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:32.987+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:32.988+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:32.989+0000] {subprocess.py:93} INFO - 25/08/19 09:28:32 INFO TaskSchedulerImpl: Removed TaskSet 4.0, whose tasks have all completed, from pool
[2025-08-19T09:28:33.268+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 WARN TaskSetManager: Lost task 1.0 in stage 2.0 (TID 5) (172.18.0.12 executor 0): TaskKilled (Stage cancelled: Job 2 cancelled part of cancelled job tag broadcast exchange (runId 1500a951-a34b-4157-a998-3dee31bb4d7e))
[2025-08-19T09:28:33.283+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 WARN TaskSetManager: Lost task 0.0 in stage 3.0 (TID 6) (172.18.0.12 executor 0): TaskKilled (Stage cancelled: Job 3 cancelled part of cancelled job tag broadcast exchange (runId 56c3bfb1-92c2-47af-aaff-6113d5d0d318))
[2025-08-19T09:28:33.308+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 WARN TaskSetManager: Lost task 0.0 in stage 2.0 (TID 4) (172.18.0.12 executor 0): TaskKilled (Stage cancelled: Job 2 cancelled part of cancelled job tag broadcast exchange (runId 1500a951-a34b-4157-a998-3dee31bb4d7e))
[2025-08-19T09:28:33.312+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 INFO TaskSchedulerImpl: Removed TaskSet 2.0, whose tasks have all completed, from pool
[2025-08-19T09:28:33.324+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 WARN TaskSetManager: Lost task 1.0 in stage 3.0 (TID 7) (172.18.0.12 executor 0): TaskKilled (Stage cancelled: Job 3 cancelled part of cancelled job tag broadcast exchange (runId 56c3bfb1-92c2-47af-aaff-6113d5d0d318))
[2025-08-19T09:28:33.327+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 INFO TaskSchedulerImpl: Removed TaskSet 3.0, whose tasks have all completed, from pool
[2025-08-19T09:28:33.600+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2025-08-19T09:28:33.604+0000] {subprocess.py:93} INFO -   File "/opt/***/jobs/transformation/transform_to_golden_dataset.py", line 135, in <module>
[2025-08-19T09:28:33.607+0000] {subprocess.py:93} INFO -     final_df.write.format("parquet").mode("overwrite").save("hdfs://namenode:9000/storage/hdfs/processed/golden_dataset")
[2025-08-19T09:28:33.609+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/pyspark.zip/pyspark/sql/readwriter.py", line 1463, in save
[2025-08-19T09:28:33.612+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2025-08-19T09:28:33.612+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 179, in deco
[2025-08-19T09:28:33.613+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2025-08-19T09:28:33.617+0000] {subprocess.py:93} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o1969.save.
[2025-08-19T09:28:33.618+0000] {subprocess.py:93} INFO - : org.apache.spark.SparkException: Job aborted due to stage failure: Task 1 in stage 0.0 failed 4 times, most recent failure: Lost task 1.3 in stage 0.0 (TID 26) (172.18.0.17 executor 1): org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00011-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:33.619+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:33.620+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:33.621+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:33.622+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:33.623+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:33.625+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:33.626+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-08-19T09:28:33.627+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:33.628+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:33.629+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:33.631+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:33.632+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:33.633+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:33.634+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:33.635+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:33.636+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:33.636+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:33.637+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:33.638+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:33.639+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:33.640+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:33.641+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:33.642+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:33.643+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:33.644+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:33.645+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:33.646+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:33.646+0000] {subprocess.py:93} INFO - Driver stacktrace:
[2025-08-19T09:28:33.647+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.failJobAndIndependentStages(DAGScheduler.scala:2856)
[2025-08-19T09:28:33.647+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2(DAGScheduler.scala:2792)
[2025-08-19T09:28:33.647+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2$adapted(DAGScheduler.scala:2791)
[2025-08-19T09:28:33.648+0000] {subprocess.py:93} INFO - 	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
[2025-08-19T09:28:33.649+0000] {subprocess.py:93} INFO - 	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
[2025-08-19T09:28:33.650+0000] {subprocess.py:93} INFO - 	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
[2025-08-19T09:28:33.654+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.abortStage(DAGScheduler.scala:2791)
[2025-08-19T09:28:33.655+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1(DAGScheduler.scala:1247)
[2025-08-19T09:28:33.656+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1$adapted(DAGScheduler.scala:1247)
[2025-08-19T09:28:33.657+0000] {subprocess.py:93} INFO - 	at scala.Option.foreach(Option.scala:407)
[2025-08-19T09:28:33.657+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.handleTaskSetFailed(DAGScheduler.scala:1247)
[2025-08-19T09:28:33.659+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.doOnReceive(DAGScheduler.scala:3060)
[2025-08-19T09:28:33.660+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2994)
[2025-08-19T09:28:33.660+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2983)
[2025-08-19T09:28:33.661+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.EventLoop$$anon$1.run(EventLoop.scala:49)
[2025-08-19T09:28:33.661+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.DAGScheduler.runJob(DAGScheduler.scala:989)
[2025-08-19T09:28:33.661+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2398)
[2025-08-19T09:28:33.661+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2419)
[2025-08-19T09:28:33.662+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2438)
[2025-08-19T09:28:33.663+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.runJob(SparkContext.scala:2463)
[2025-08-19T09:28:33.664+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$collect$1(RDD.scala:1049)
[2025-08-19T09:28:33.665+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:151)
[2025-08-19T09:28:33.666+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:112)
[2025-08-19T09:28:33.667+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.withScope(RDD.scala:410)
[2025-08-19T09:28:33.670+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.collect(RDD.scala:1048)
[2025-08-19T09:28:33.671+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.executeCollectIterator(SparkPlan.scala:455)
[2025-08-19T09:28:33.671+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.exchange.BroadcastExchangeExec.$anonfun$relationFuture$1(BroadcastExchangeExec.scala:140)
[2025-08-19T09:28:33.671+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withThreadLocalCaptured$2(SQLExecution.scala:224)
[2025-08-19T09:28:33.672+0000] {subprocess.py:93} INFO - 	at org.apache.spark.JobArtifactSet$.withActiveJobArtifactState(JobArtifactSet.scala:94)
[2025-08-19T09:28:33.672+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withThreadLocalCaptured$1(SQLExecution.scala:219)
[2025-08-19T09:28:33.672+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2025-08-19T09:28:33.672+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2025-08-19T09:28:33.673+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2025-08-19T09:28:33.674+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-19T09:28:33.675+0000] {subprocess.py:93} INFO - Caused by: org.apache.spark.SparkFileNotFoundException: File does not exist: hdfs://namenode:9000/storage/hdfs/raw/avro/KR_category_id/part-00011-2fbd0631-ab75-4527-939a-099e8c4bcb15-c000.avro
[2025-08-19T09:28:33.676+0000] {subprocess.py:93} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2025-08-19T09:28:33.676+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:780)
[2025-08-19T09:28:33.677+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2025-08-19T09:28:33.678+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2025-08-19T09:28:33.680+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2025-08-19T09:28:33.681+0000] {subprocess.py:93} INFO - 	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2025-08-19T09:28:33.682+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage7.processNext(Unknown Source)
[2025-08-19T09:28:33.683+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2025-08-19T09:28:33.684+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2025-08-19T09:28:33.684+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2025-08-19T09:28:33.684+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2025-08-19T09:28:33.685+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2025-08-19T09:28:33.685+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2025-08-19T09:28:33.685+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2025-08-19T09:28:33.685+0000] {subprocess.py:93} INFO - 	at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2025-08-19T09:28:33.686+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2025-08-19T09:28:33.687+0000] {subprocess.py:93} INFO - 	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2025-08-19T09:28:33.688+0000] {subprocess.py:93} INFO - 	at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2025-08-19T09:28:33.689+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2025-08-19T09:28:33.689+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2025-08-19T09:28:33.690+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2025-08-19T09:28:33.691+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2025-08-19T09:28:33.692+0000] {subprocess.py:93} INFO - 	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2025-08-19T09:28:33.693+0000] {subprocess.py:93} INFO - 	... 3 more
[2025-08-19T09:28:33.694+0000] {subprocess.py:93} INFO - 
[2025-08-19T09:28:33.744+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 INFO SparkContext: Invoking stop() from shutdown hook
[2025-08-19T09:28:33.760+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2025-08-19T09:28:33.789+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 INFO SparkUI: Stopped Spark web UI at http://b208641feb52:4040
[2025-08-19T09:28:33.808+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 INFO StandaloneSchedulerBackend: Shutting down all executors
[2025-08-19T09:28:33.809+0000] {subprocess.py:93} INFO - 25/08/19 09:28:33 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2025-08-19T09:28:34.148+0000] {subprocess.py:93} INFO - 25/08/19 09:28:34 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2025-08-19T09:28:34.213+0000] {subprocess.py:93} INFO - 25/08/19 09:28:34 INFO MemoryStore: MemoryStore cleared
[2025-08-19T09:28:34.222+0000] {subprocess.py:93} INFO - 25/08/19 09:28:34 INFO BlockManager: BlockManager stopped
[2025-08-19T09:28:34.341+0000] {subprocess.py:93} INFO - 25/08/19 09:28:34 INFO BlockManagerMaster: BlockManagerMaster stopped
[2025-08-19T09:28:34.378+0000] {subprocess.py:93} INFO - 25/08/19 09:28:34 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2025-08-19T09:28:34.477+0000] {subprocess.py:93} INFO - 25/08/19 09:28:34 INFO SparkContext: Successfully stopped SparkContext
[2025-08-19T09:28:34.478+0000] {subprocess.py:93} INFO - 25/08/19 09:28:34 INFO ShutdownHookManager: Shutdown hook called
[2025-08-19T09:28:34.489+0000] {subprocess.py:93} INFO - 25/08/19 09:28:34 INFO ShutdownHookManager: Deleting directory /tmp/spark-4efd3942-35f0-4efc-871e-ce789dbc7c41/pyspark-9a1c2365-fb81-4776-8351-2fad7a0bb468
[2025-08-19T09:28:34.521+0000] {subprocess.py:93} INFO - 25/08/19 09:28:34 INFO ShutdownHookManager: Deleting directory /tmp/spark-ef2aab3f-b00e-42fa-a09e-b0cb51c4e209
[2025-08-19T09:28:34.536+0000] {subprocess.py:93} INFO - 25/08/19 09:28:34 INFO ShutdownHookManager: Deleting directory /tmp/spark-4efd3942-35f0-4efc-871e-ce789dbc7c41
[2025-08-19T09:28:34.988+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-08-19T09:28:34.990+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-19T09:28:35.055+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-08-19T09:28:35.076+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=youtube_pipeline_batch_data, task_id=transform_golden, execution_date=20250819T091107, start_date=20250819T091726, end_date=20250819T092835
[2025-08-19T09:28:35.129+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 597 for task transform_golden (Bash command failed. The command returned a non-zero exit code 1.; 2219)
[2025-08-19T09:28:35.180+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-08-19T09:28:35.276+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-19T09:28:35.285+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
